package.json

```diff
@@ -18,6 +18,6 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "devDependencies": {
-    "semantic-release": "^6.3.2"
+    "semantic-release": "^15.10.5"
```

Comment:
If we're going to upgrade this dependency, then we need to update the `semantic-release` script because there were breaking changes. It should be simply: `"semantic-release": "semantic-release"` :)

Reply:
I did not quite understand what I meant when I said "semantic-release": "semantic-release" would be in package.json? Omitting the version? Can you make it clearer? I can do this task immediately.